TrueCraft.Launcher/Views/MultiplayerView.cs

```diff
@@ -131,7 +143,10 @@ public MultiplayerView(LauncherWindow window)
             foreach (var server in UserSettings.Local.FavoriteServers)
             {
                 var row = ServerListStore.AddRow();
-                using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
+                using (
+                    var stream =
+                        Assembly.GetExecutingAssembly()
+                            .GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
```

Comment:
That would be clear if I pushed it, doesn't it count for something that I
created a PR? I guess you really don't get it, too bad
On Jun 9, 2015 00:47, "Drew DeVault" <notifications@github.com> wrote:

> In TrueCraft.Launcher/Views/MultiplayerView.cs
> <https://github.com/SirCmpwn/TrueCraft/pull/116#discussion_r31967989>:
>
> > @@ -131,7 +143,10 @@ public MultiplayerView(LauncherWindow window)
> >              foreach (var server in UserSettings.Local.FavoriteServers)
> >              {
> >                  var row = ServerListStore.AddRow();
> > -                using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
> > +                using (
> > +                    var stream =
> > +                        Assembly.GetExecutingAssembly()
> > +                            .GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
>
> It's clear that it was a mistake to give you write access to this
> repository.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SirCmpwn/TrueCraft/pull/116/files#r31967989>.
>